### PR TITLE
Improve wallets documentation

### PR DESCRIPTION
**Pull request by martonlederer**
- Modify links to point to the main wallets FAQ homepage
- Add separate guide links to Arweave.app ~~and ArConnect~~
- Move Arweave.app FAQ questions to it's guide page
~~- Add guide to ArConnect~~

### CODE REVIEW

File: wallets/arweave-wallet.md [hunk 1]
```diff
@@ -84,4 +84,10 @@ To remove a set of wallet keys from the wallet app, click the <img src="../.gitb
 
 From this screen you'll be able to re-download the key file for a particular wallet key pair or click the "Delete" button to remove it from the wallet app entirely. As long as you have your passphrase or key file you'll be able to add it back whenever you like so keep those safe and secure.
 
+## **Using Arweave.app as a cold wallet**
```
Comment (rosmcmahon) [inline wallets/arweave-wallet.md:87]: this needs to stay in FAQs. it's the most popular question, and has even got it's own page.

File: wallets/wallet-faq.md [hunk 2]
```diff
@@ -8,17 +8,66 @@ description: All questions surrounding wallets
...
-## **How do I use the arweave.app web wallet?**
-
-Lucky for you, we’ve created a step-by-step guide which can be [**found here**](https://docs.arweave.org/info/wallets/arweave-web-extension-wallet)**.**
-
-## **Can I use the arweave.app web wallet as a cold wallet?**
-
-Yes, you can find detailed instructions for how to securely generate an Arweave cold wallet with the arweave.app [**here**](https://docs.arweave.org/info/wallets/generating-cold-wallet).
-
-**\* Please make sure to keep multiple copies of your wallet key file in secure locations, as we cannot help you to recover your wallet if you lose it \***
+<table>
```
Comment (rosmcmahon) [inline wallets/wallet-faq.md:11]: these FAQs need to stay.

File: wallets/wallet-faq.md [hunk 3]
```diff
@@ -8,17 +8,66 @@ description: All questions surrounding wallets
...
-
-## **How do I use the arweave.app web wallet?**
-
-Lucky for you, we’ve created a step-by-step guide which can be [**found here**](https://docs.arweave.org/info/wallets/arweave-web-extension-wallet)**.**
-
-## **Can I use the arweave.app web wallet as a cold wallet?**
-
-Yes, you can find detailed instructions for how to securely generate an Arweave cold wallet with the arweave.app [**here**](https://docs.arweave.org/info/wallets/generating-cold-wallet).
-
-**\* Please make sure to keep multiple copies of your wallet key file in secure locations, as we cannot help you to recover your wallet if you lose it \***
```
Comment (rosmcmahon) [inline wallets/wallet-faq.md:21]: what is the benefit to users of removing this?

**Comment by rosmcmahon** [comment]
DMed you

**Comment by martonlederer** [comment]
Fixed up issues:
- Added back content to wallet FAQ (+improvements and extra FAQ)
- Added guides for generating cold wallets with ArConnect and the ArDrive CLI (the ArDrive CLI is considered advanced, so I added a note about this in the beginning of the guide)
- Added images for the cold wallet generation guide (both for Arweave.app and ArConnect)
- Finished up the ArConnect guide (setup section, disclaimer about stored wallets, manage wallets section, settings section)

**Comment by martonlederer** [comment]
- Moved ArConnect docs to #20

**Comment by rosmcmahon** [comment]
as discussed this PR will not be merged.

**Pull request closed without merging**